src/parsers/power_system_table_data.jl

```diff
@@ -266,13 +266,14 @@ function System(
 
     loadzone_csv_parser!(sys, data)
     bus_csv_parser!(sys, data)
-    load_csv_parser!(sys, data)
+    busload_csv_parser!(sys, data)
```

Comment:
It might just be me, but I don't find it obvious as to why you create PowerLoads implicitly from buses and explicitly from raw data.  Is any of this redundant?  Does it merit a comment here?

Reply:
I don't know what the right approach here is. In this instance, I've added the explicit load calculation because I want to do both. I have a dataset that has load's defined as implicit loads at each bus, and a seperate loads.csv defining auxillary loads. This may be an edge case that we don't want to support directly. I think the common cases that we should support are:
 1. bus.csv contains the implicit load
 2. bus.csv doesn't contain implicit loads, all loads are contained in load.csv

Reply:
How about this?

You could look for loads in load.csv.  If you find some, don't create them implicitly from buses.  Allow an override flag to always create implicit loads.